### ISSUE TITLE
feature request: fortran support in hdf5

### ISSUE THREAD

**Issue by AlexanderKoshkarov** [voidlinux/void-packages#13894]
A lot of scientific softwear use hdf5 and fortran :)
So if you would not mind, please add "--enable-fortran" flag to hdf5 package

**Comment by Piraty** [comment]
i prepared the neccesary steps, see the mentioned PR.
@AlexanderKoshkarov which packages would need a rebuild?

**Comment by AlexanderKoshkarov** [comment]
Thank you!
you mean what packages in void repos depend on this feature? Well, there are a lot of scientific softwear which require hdf5, but not in the repos (paraview, visit, to mention big ones) and a lot of simulation codes. Another thing which depends on hdf5 is another common format netcdf. I feel that I do not fully understand your question :(

**Comment by Piraty** [comment]
Sure. Some packages depend on hdf5, but so far worked fine without the requested change.
I rephrase: Is there any of void's packages in particular you would like to see rebuilt against the fortran-enabled hdf5 to make use of the new setting?

**Comment by AlexanderKoshkarov** [comment]
Not really - for my purpose I need it mostly because of a lot of scientific code which are not in repos. If it is not very difficult to rebuild, I think netcdf would be good to rebuild with new hdf5. Thank you.

**Comment by Piraty** [comment]
Indeed i looked into netcdf, but adding gcc-fortran to makedepends only satisfies a configure statement, and given the new hdf5 the build log is exactly the same. So i don't suspect improvement of anything there...

**Comment by AlexanderKoshkarov** [comment]
ah, you are right, netcdf guys separated all "fortran" into separate netcdf-fortran package.

**Comment by Piraty** [comment]
yes, this option is provded by netcdf's configure script, and marked as "experinemtal".
```
--enable-remote-fortran-bootstrap
                          Download and install netcdf-fortran (EXPERIMENTAL)
```
